Makefile

```diff
@@ -148,4 +148,4 @@ docker-generate: clean
 	sleep 3
 	@docker stop genenv || true && docker rm --force genenv
 
-.PHONY: all deps fmt install release clean check test test-long test-long-ci start-mongo stop-mongo docker-generate
+.PHONY: all deps fmt install release clean check test test-long test-long-ci test-single start-mongo stop-mongo docker-generate
```

Comment:
I'm not against new commands, I'm just not seeing the difference between this command and `test-long-ci`.

Reply:
this can be removed as a follow up.